src/main/java/bio/terra/common/tracing/OkHttpClientTracingInterceptor.java

```diff
@@ -0,0 +1,56 @@
+package bio.terra.common.tracing;
+
+import io.opencensus.contrib.http.HttpClientHandler;
+import io.opencensus.contrib.http.HttpRequestContext;
+import io.opencensus.trace.Tracer;
+import io.opencensus.trace.Tracing;
+import io.opencensus.trace.propagation.TextFormat.Setter;
+import java.io.IOException;
+import okhttp3.Interceptor;
+import okhttp3.Request;
+import okhttp3.Response;
+
+public class OkHttpClientTracingInterceptor implements Interceptor {
+  private Tracer tracer;
+  // This looks a little odd, but OkHttp requests are immutable so the carrier type must be a
+  // request builder, not a Request.
+  private HttpClientHandler<Request, Response, Request.Builder> handler;
+  private static final Setter<Request.Builder> SETTER =
+      new Setter<Request.Builder>() {
+        @Override
```

Comment:
can this be a lambda? I.e. is it a functional interface? Maybe not if there's not actually an interface. 🤔

Reply:
I don't think so because `Setter` is an abstract class and not an interface, but I'm far from sure on that.

Reply:
IJ will highlight the section if it's lambda-able.